CONTRIBUTING.md

```diff
@@ -116,12 +116,18 @@ In the dev tools you can inspect the DOM, debug your JavaScript code, etc. - as
 
 ### Automated tests
 
-Cucumber is used for end-to-end functional/acceptance testing. You should regularly run the tests to ensure that new code does not result in regressions. The tests are currently only running on Linux and macOS. To execute them you can run:
+[Jest](https://jestjs.io/docs/expect) is used for unit testing. Please add unit tests whenever is possible. For example of how mocking can be implemented see [docx_controller.spec.js](https://github.com/ezra-project/ezra-project/blob/c4957dee0c7e66e4708a1d532292ec46ff01f4fa/app/frontend/components/docx_export/__tests__/docx_controller.spec.js). For examples of rendering DOM UI elements see [loading_indicator.spec.js](https://github.com/ezra-project/ezra-project/blob/2349d6a20dfb02cef32a54efe436b31c1d2507e0/app/frontend/components/__tests__/loading_indicator.spec.js) and [assistant_checkbox.spec.js](https://github.com/ezra-project/ezra-project/blob/eb9fd583451f843597a26089ebfd28c0c2bcedd2/app/frontend/components/module_assistant/__tests__/assistant_checkbox.spec.js). DOM rendering is done through `jest-electron`, that requires lower version of `jest`.
 
-- `npm run full-test` to execute the tests in the foreground.
-- `npm run headless-test` to execute the tests in the background without showing the window (Linux-only).
-- `npm run dev-test` to execute the tests only until an error is encountered (Stop after the first error).
+[Cucumber](https://cucumber.io/docs/cucumber/) is used for end-to-end functional/acceptance testing. 
 
+You should regularly run the tests to ensure that new code does not result in regressions. The tests are currently only running on Linux and macOS. To execute them you can run:
+
+- `npm test` to execute all unit tests.
```

Comment:
We should either move the note about executing unit tests to the section about Jest or have a separate section dedicated on "executing tests".